change Shadows defaultValue from 2 to 0 to avoid related hangs/crashes
until the root cause is identified

Index: rts/Rendering/ShadowHandler.cpp
--- rts/Rendering/ShadowHandler.cpp.orig
+++ rts/Rendering/ShadowHandler.cpp
@@ -29,7 +29,8 @@
 
 #include "lib/fmt/format.h"
 
-CONFIG(int, Shadows).defaultValue(2).headlessValue(-1).minimumValue(-1).safemodeValue(-1).description("Sets whether shadows are rendered.\n-1:=forceoff, 0:=off, 1:=full, 2:=fast (skip terrain)"); //FIXME document bitmask
+// OpenBSD only: change Shadows defaultValue from 2 to 0 to avoid related hangs/crashes
+CONFIG(int, Shadows).defaultValue(0).headlessValue(-1).minimumValue(-1).safemodeValue(-1).description("Sets whether shadows are rendered.\n-1:=forceoff, 0:=off, 1:=full, 2:=fast (skip terrain)"); //FIXME document bitmask
 CONFIG(int, ShadowMapSize).defaultValue(CShadowHandler::DEF_SHADOWMAP_SIZE).minimumValue(32).description("Sets the resolution of shadows. Higher numbers increase quality at the cost of performance.");
 CONFIG(int, ShadowProjectionMode).defaultValue(CShadowHandler::SHADOWPROMODE_CAM_CENTER);
 CONFIG(bool, ShadowColorMode).defaultValue(true).description("Whether the colorbuffer of shadowmap FBO is RGB vs greyscale(to conserve some VRAM)");
